### PR TITLE
docs(readme): llm-generative-art revamp

### PR DESCRIPTION
**Pull request by alexcg1**
Revamp of readme to focus more on LLMs and generative stuff

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: README-bullet-short.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+
+<!-- start jina-description -->
+
+Jina lets you build multimodal [**AI services**](#build-ai-services) and [**pipelines**](#build-a-pipeline) that communicate via gRPC, HTTP and WebSockets, then scale them up and deploy to production. You can focus on your logic and algorithms, without worrying about the infrastructure complexity.
+
+Jina provides a smooth Pythonic experience transitioning from local deployment to advanced orchestration frameworks like Docker-Compose, Kubernetes, or Jina AI Cloud. Jina makes advanced solution engineering and cloud-native technologies accessible to every developer.
+
+Applications built with Jina enjoy the following features out of the box:
+
+- Build applications for any [data type](), any mainstream [deep learning framework](), and any [gateway]().
```
[the remark this answers is not in the document]
Comment (alexcg1) [reply README-bullet-short.md:33]: I'll change to "popular gateways" or "commonly-used gateways"

[the remark this answers is not in the document]
Comment (JoanFM) [reply README-bullet-short.md:33]: But what do u mean here?

[the remark this answers is not in the document]
Comment (JoanFM) [reply README-bullet-short.md:33]: okey, protocols

[the remark this answers is not in the document]
Comment (alexcg1) [reply README-bullet-short.md:33]: Fixed. In README.md, not this short-bullet form

**Comment by alexcg1** [comment]
Note, I'm completely removing the notebook because:

- StableLM Executor in Deployment won't work in Colab due to GPU/memory constraints
- TextToImage Executor in Flow won't work because Hub Executors won't be sharable/reusable soon
- As @samsja said, Jina is seldom used in notebooks anyway
- With DocArray v2 as default, we'd have to refactor anyway

**Comment by alexcg1** [comment]
For https://github.com/jina-ai/jina/pull/5895/commits/64fc9cc46f03e47a78f881237bff8fed7d4278a7, I grepped the repo to ensure the images didn't appear anywhere.

I assume they're not called from any external places, right?

**Comment by alexcg1** [comment]
@alaeddine-13 @JoanFM can you review? I think I've made all the big changes now

File: README.md [hunk 2]
```diff
@@ -25,41 +23,19 @@
...
-  - Full observability via OpenTelemetry, Prometheus and Grafana.
-  - Fast deployment to Kubernetes and Docker Compose.
-
-🍱 **Ecosystem**
-  - Improved engineering efficiency thanks to the Jina AI ecosystem, so you can focus on innovating with the data applications you build.
-  - Free CPU/GPU hosting via [Jina AI Cloud](https://cloud.jina.ai).
+- Build applications for any [data type](https://docs.docarray.org/data_types/first_steps/), any mainstream [deep learning framework](), and any [protocol](https://docs.jina.ai/concepts/serving/gateway/#set-protocol-in-python).
+- Design high-performance microservices, with [easy scaling](https://docs.jina.ai/concepts/orchestration/scale-out/), duplex client-server streaming, and async/non-blocking data processing over dynamic flows.
+- Docker container integration via [Executor Hub](https://cloud.jina.ai), OpenTelemetry/Prometheus observability, and fast Kubernetes/Docker-Compose deployment.
+- Free CPU/GPU hosting via [Jina AI Cloud](https://cloud.jina.ai).
```
Comment (JoanFM) [inline README.md:35]: This is not Free

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alexcg1) [reply README.md]: Fixed

**Review by JoanFM — APPROVED** [review]

**Pull request merged**